### PR TITLE
chore(ci): remove snyk jobs as we are decommissioning it

### PR DESCRIPTION
**Pull request by jjacque**
### What does this PR do?

Remove Snyk analysis from CI

### Motivation

It's too expensive ;)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 version: 2.1
-orbs:
-  snyk: snyk/snyk@2.0.1
```
Comment (v4ld3r5) [inline .circleci/config.yml:3]: do we need to provide a new orb here? or will this cause a failover to a default one?

Comment (v4ld3r5) [reply .circleci/config.yml:3]: maybe this? 
```yaml
orbs:
  golang: circleci/go@1.10.0
```

Comment (jjacque) [reply .circleci/config.yml:3]: actually we won't use Snyk at all anymore and i'm not aware of a new static code analysis tool validated yet.

**Review by v4ld3r5 — APPROVED** [review]
lgtm

**Pull request merged**